pyirf/statistics.py

```diff
@@ -32,8 +32,11 @@ def li_ma_significance(n_on, n_off, alpha=0.2):
 
     scalar = is_scalar(n_on)
 
-    n_on = np.array(n_on, copy=False, ndmin=1)
-    n_off = np.array(n_off, copy=False, ndmin=1)
+    # Cast everything into float64 to avoid numeric instabilties
+    # when multiplying very small and very big numbers to get t1 and t2
+    n_on = np.array(n_on, copy=False, ndmin=1, dtype=np.float64)
+    n_off = np.array(n_off, copy=False, ndmin=1, dtype=np.float64)
+    alpha = np.float64(alpha)
```

Comment:
This prevents broadcasting over alpha, right?

Reply:
Probably, but looking at the docsting this was never intended

Reply:
Right. Can you fix the typo (integer of → integer or) ?